frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -183,13 +183,15 @@ const Profile: NextPage = () => {
   const subdomainMessage =
     typeof profile.subdomain === "string" ? (
       <>
-        {l10n.getString("profile-label-domain")}&nbsp;
-        <SubdomainIndicator
-          subdomain={profile.subdomain}
-          onCreateAlias={(address: string) =>
-            createAlias({ type: "custom", address: address })
-          }
-        />
+        <span>{l10n.getString("profile-label-domain")}</span>
```

Comment:
This works and is totally fine, but I thought you might appreciate hearing about the [`<wbr>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr) element (I did, when I first heard about it) :slightly_smiling_face:

Reply:
Whoa! I have not heard about `wbr`. This is fantastic! I tried to refactor my fix, but still wasn't getting the actual "mydomain.mozmail.com" to break without the extra `word-break: break-word;` being applied. Thanks for the TIL! 

![image](https://user-images.githubusercontent.com/2692333/165986746-7fb2bc66-3952-473d-8a45-6b1dc7014aac.png)